src/Symfony/Component/Console/Terminal.php

```diff
@@ -87,25 +87,13 @@ private static function initDimensions()
      */
     private static function getConsoleMode()
     {
-        if (!\function_exists('proc_open')) {
```

Comment:
💡 This piece of code was duplicated. Before fixing it twice, I've extracted it into a method.